Entity/Concrete/Product.cs

```diff
@@ -7,8 +8,15 @@ public class Product : BaseEntity
     public Guid CategoryId { get; set; }
     public Guid SupplierId { get; set; }
 
+    [Required(AllowEmptyStrings = false, ErrorMessage = "Please enter a name")]
```

Comment:
I thought we were using fluent validator for server side or am I missing something?

Reply:
> Well, usually there is only single library used for validation to avoid code mixing with different styles. Let's use JQuery here instead

https://github.com/osmanaliyardim/WebNetSample/pull/3/commits/7b9c7e45fd513de9fe73a05c4d669f9f962b06c6